reporting_app/static/plate_view.js

```diff
@@ -159,6 +130,7 @@ function render_heatmap(colour_metric) {
                 function(data_point) {
                     var datatype = typeof data_point.value;
                     if (datatype == 'object') {
+                        console.log(data_point.value)
                         data_point.value = data_point.value.sort().join(', ');
```

Comment:
When I select Genotype match on the genotypes heatmap, I'm getting TypeErrors here ('data_point.value is null') and it's showing up as a number - surely this should be a category (match, mismatch, unknown)? Looks like it's crashing and still showing the data from the last selected metric but presenting it as genotype match.

Reply:
That should now be fixed